chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb

```diff
@@ -205,6 +205,14 @@
   action :create
 end
 
+template "/usr/sbin/crowbar-dhcp-migration.sh" do
```

Comment:
It might be better to just put this file on the controller nodes and not everywhere (but I know reouter scripts are also everywhere...)